web/src/lib/llm/utils.ts

```diff
@@ -69,20 +69,20 @@ const MODEL_NAMES_SUPPORTING_IMAGE_INPUT = [
   "gpt-4-turbo",
   "gpt-4-1106-vision-preview",
   // standard claude names
-  "claude-3-5-sonnet-20240620",
+  "claude-3-5-sonnet-20241022",
   "claude-3-opus-20240229",
   "claude-3-sonnet-20240229",
   "claude-3-haiku-20240307",
```

Comment:
Users that have previously set their default model should still have access to image input